addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,118 @@
+# iCloud Binding
+The Apple [iCloud](https://www.apple.com/icloud/) is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account. Updates are quick and accurate without significant battery time impact. The Binding also offers access to the "Find My iPhone" function.
```

Comment:
Please add empty lines after any header and before and after any code section.

Reply:
Please also add a line break after every sentence.

Reply:
Done.